package.json

```diff
@@ -18,8 +18,8 @@
   },
   "homepage": "https://github.com/18F/identity-dev-docs#readme",
   "dependencies": {
-    "@18f/identity-build-sass": "^1.1.0",
-    "@18f/identity-design-system": "^7.1.0"
+    "@18f/identity-build-sass": "^3.0.0",
```

Comment:
One of the features included in this upgrade (specifically [v1.2.0](https://github.com/18F/identity-idp/blob/main/app/javascript/packages/build-sass/CHANGELOG.md#120)) can allow you to drop the `--load-path` from the `build:css` script above:

```diff
diff --git a/package.json b/package.json
index 3fdd9aa..db8668a 100644
--- a/package.json
+++ b/package.json
@@ -4,5 +4,5 @@
   "scripts": {
     "prebuild:css": "mkdir -p _site/assets/css",
-    "build:css": "build-sass assets/scss/*.scss --out-dir=_site/assets/css --load-path=node_modules/@18f/identity-design-system/packages",
+    "build:css": "build-sass assets/scss/*.scss --out-dir=_site/assets/css",
     "watch:css": "npm run build:css -- --watch",
     "pages": "npm run build:css",
```

Reply:
Removing it! Thanks for the heads up.